src/coreComponents/mesh/utilities/ComputationalGeometry.hpp

```diff
@@ -79,8 +79,8 @@ void LinePlaneIntersection( LINEDIR_TYPE const & lineDir,
  * @param[in] normal vector normal to the plane
  */
 template< typename NORMAL_TYPE >
-void orderPointsCCW( arrayView2d< real64 > const & points,
-                     NORMAL_TYPE const & normal )
+std::vector< int >  orderPointsCCW( arrayView2d< real64 > const & points,
```

Comment:
does this have to be `std::vector`?

Reply:
TBH, it 's an `std::vector` because that's what was used inside the function and I just made it return that object since I needed it. I suspect that the function uses it to be able to use :
```
 // sort the indices
  std::sort( indices.begin(), indices.end(), [&]( int i, int j ) { return angle[ i ] < angle[ j ]; } );
```
If you want I can copy those values in `array1d` and return that. This should only be executed on host though (if that's your concern).

Reply:
That same call works with an `array1d`. But for host only temporary stuff std::vector is fine by me.

Reply:
I made it an `array1d`.